webpack.config.js

```diff
@@ -124,6 +124,9 @@ module.exports = (_, { mode }) => {
     },
     resolve: {
       extensions: ['.js', '.jsx', '.json'],
+      alias: {
+        react: path.resolve('./node_modules/react'),
```

Comment:
What is this for? Aliasing is usually done when you want to replace imports of say `react` with `preact`...

Reply:
making sure to use the correct react when there could posibly be 2 diff versions - i had an issue with useEffect aka 16.8+ react and another verison. `npm ls react` didnt show me any differences but this was the only solution that worked for me